frontend/src/metabase/selectors/data.unit.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { State } from "metabase-types/store";
+import {
+  getHasDataAccess,
+  getHasOwnDatabase,
+  getHasNativeWrite,
+  getHasDatabaseWithJsonEngine,
+} from "./data";
+
+describe("metabase/selectors/data", () => {
+  describe("getHasDataAccess", () => {
+    it("should return true if user has data access", () => {
+      const state = {
+        entities: {
+          databases: { databaseOne: { is_saved_questions: false } },
+        },
+      };
+
+      expect(getHasDataAccess(state as any)).toBe(true);
+    });
+
+    it("should return false if user does not have data access", () => {
```

Comment:
WDYT about giving a bit more context in test titles? The connection between "data access" and a set of databases in the `entities` reducer might not be clear. Should the test say we expect at least one real database connected for the selector to return `true`?

Reply:
Yes, better.

Reply:
@kulyk I also removed this "should return true" part from the titles. 

Basically I had reused this test file

https://github.com/metabase/metabase/blob/e7b67637f87313887d6204ac3635ff5eb3c31069/frontend/src/metabase/selectors/user.unit.spec.js

as the source of the approach, and now it looks like that file is the one needing a bit of love.